modules/core/src/test/scala/doobie/util/FragmentsSuite.scala

```diff
@@ -25,87 +25,87 @@ class FragmentsSuite extends munit.FunSuite {
   val ofs  = List(1,2,3).map(n => Some(fr"$n").filter(_ => n % 2 =!= 0))
 
   test("values for one column") {
-    assertEquals(values(nel).query[Unit].sql, "VALUES (?), (?), (?) ")
+    assertEquals(values(nel).query[Unit].sql, "VALUES ?, ?, ? ")
```

Comment:
Hmm I think this is a breaking change. I think the parenthesis is always needed?

https://www.db-fiddle.com/f/UQ1QzHQvkj1niuynfP4ob/0

Reply:
Of course! You're correct, I've fixed it. Thank you